src/lapack/util/app/qut/front/flamec/test/flash_sm/REF_Apply_Q.c

```diff
@@ -12,6 +12,6 @@
 
 FLA_Error REF_Apply_Q( FLA_Side side, FLA_Uplo uplo, FLA_Trans trans, FLA_Obj A, FLA_Obj t, FLA_Obj B )
 {
-  return FLA_Apply( side, uplo, trans, A, t, B );
+  //return FLA_Apply( side, uplo, trans, A, t, B );
```

Comment:
No reference to this. However, not having it seems to be causing a runtime issue. Will keep looking into it.